lab_1/main.py

```diff
@@ -12,8 +11,17 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    try:
```

Comment:
`try ... except` blocks are usually wrap the line where the Exception is likely to be raised.

Take a look at `isinstance` for type-checking.